test/test_log.py

```diff
@@ -126,6 +140,30 @@ def test_reduced_logging(self):
         logger.warning('warning')
         self.assertEqual(self.backend.calls(), [('WARNING', 'warning')])
 
+    def test_long_string_logging(self):
+        buffer = io.StringIO()
+
+        with patch('sys.stderr', buffer):
+            ops.log.setup_root_logging(self.backend, debug=True)
+            logger = logging.getLogger()
+            logger.debug('{}'.format('l' * 131071))
+
+        self.assertEqual(len(self.backend.calls()), 1)
+
+        self.backend.calls(clear=True)
+
+        with patch('sys.stderr', buffer):
+            logger.debug('{}'.format('l' * 131080))
```

Comment:
```suggestion
            logger.debug('{}'.format('l' * (MAX_STR_LEN + 9))
```

Reply:
Not committed to `+9`, but feels like we should use the defined constant to make this test resilient to any future change